package.json

```diff
@@ -178,7 +179,8 @@
     "react-test-renderer": "^16.8.6",
     "sinon": "^7.3.2",
     "url-loader": "^1.1.2",
-    "webpack-bundle-analyzer": "^3.3.2"
+    "webpack-bundle-analyzer": "^3.3.2",
+    "webpack-pwa-manifest": "^4.0.0"
```

Comment:
What is this used for? Can that be removed?

Reply:
Yes, it is unused. I will remove it.